R/addRank.R

```diff
@@ -1,60 +1,51 @@
-#' Rank the sample information in a `SummarizedExperiment` Object
+#' Rank the sample information in a `TreeSummarizedExperiment` Object
```

Comment:
in fact this could be `SE` because it is more generic than `TreeSE` (TreeSE is an SE), and this function works regardless of tree availability.